kusto/ingest/ingest.go

```diff
@@ -356,11 +356,11 @@ func (i *Ingestion) FromFile(ctx context.Context, fPath string, options ...FileO
 		}
 	}
 
-	if _, err := url.ParseRequestURI(fPath); err == nil {
-		return i.fs.Blob(ctx, fPath, 0, props)
+	if filesystem.IsFileSystem(fPath) {
```

Comment:
I considered this as well, but a shared file system path is not local, and yet it can be opened via a filesystem API.
E.g. \\host\drive\folder\path if not local, but it is a file system path

Reply:
I think that is probably getting too literal.  For our purposes, it is a local path because it is hosted locally as a drive, regardless if the mount id a local disk, remote array, NFS or CIFS share, .....  FileSystem, however, is too generic.  Everything is a filesystem, including blob store.  

LocalPath indicates it is mounted on the user system and in that case we use our selected method.